Server/mods/deathmatch/logic/CStaticFunctionDefinitions.h

```diff
@@ -700,6 +700,7 @@ class CStaticFunctionDefinitions
     // Account set funcs
     static CAccount* AddAccount(const SString& strName, const SString& strPassword, bool bAllowCaseVariations, SString& strOutError);
     static bool      RemoveAccount(CAccount* pAccount);
+    static bool      SetAccountSerial(const std::string& name, const std::string& serial) noexcept;
```

Comment:
To make the OOP syntax work, use element instead.
```suggestion
    static bool      SetAccountSerial(CAccount* account, const std::string& serial) noexcept;
```

Reply:
adjusted 7418e4c